### ISSUE TITLE
Ribbon Menu Text Spacing Issue with MahApps

### ISSUE THREAD

**Issue by chrisklepeis** [fluentribbon/Fluent.Ribbon#271]
![capture](https://cloud.githubusercontent.com/assets/16291940/13088761/934342b0-d4bb-11e5-9d61-e18ae34bc80a.PNG)

Anyone know what I would need to change in the style so that the text isn't cut off? I'm using Fluent Ribbon with MahApps. Font size maybe?

```
<ResourceDictionary Source="/Fluent;Component/Themes/Office2013/Generic.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Colors.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/Blue.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/BaseLight.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/FlatButton.xaml" />
```

**Comment by chrisklepeis** [comment]
Looks like if I set FontSize to 11 on the Ribbon itself, that works, the spacing on the tabs is a bit off though, but no big deal with that.

**Comment by batzen** [comment]
You don't have to reduce the font size as soon as i release version 4.0 of this library, there was a style bug which caused this.

**Comment by chrisklepeis** [comment]
Awesome, thanks.

I noticed one issue with the pre-release. Since I'm using MahApps, I allow users to switch the theme, using hte MahApps themes. When a user changes the theme, I set this:

`Application.Current.Resources["RibbonThemeColorBrush"] = new SolidColorBrush(Colors.Red);`

So that fluent ribbon follows the selected theme. With the prerelease, the main menu button does change color properly, but the backstage menu background doesn't. Maybe I'm trying to change the theme at runtime the wrong way?

Thanks for all your hard work on this project, it's much appreciated.
